grouper/fe/templates/service.html

```diff
@@ -0,0 +1,318 @@
+{% extends "base.html" %}
+{% from 'macros/ui.html' import group_panel, member_panel, permission_panel, log_entry_panel,
+                                help_for, account, public_key_panel, shell_panel, tokens_panel, %}
+
+{% block heading %}
+    <a href="/svc">Service Accounts</a>
+{% endblock %}
+
+{% block subheading %}
+    {{ user.username }}
+    {% if user.role_user %}<small>(role)</small>{% endif %}
+    {% if not user.enabled %}<small>(disabled)</small>{% endif %}
+
+    {% if group.audit and not group.audit.complete %}
```

Comment:
Hm, hadn't thought about how this'll interact with audits… is an audit of a SA really meaningful? I could imagine wanting to have a SA audit process, but I think for now we'll want to exclude SA groups from audits